apps/ui/src/components/Ui/LinkPreview.vue

```diff
@@ -56,7 +72,7 @@ debouncedWatch(
 
 <template>
   <div
-    v-if="preview?.meta || (showDefault && !previewLoading)"
+    v-if="preview?.meta.title || (showDefault && !previewLoading)"
```

Comment:
```suggestion
    v-if="preview?.meta?.title || (showDefault && !previewLoading)"
```

GIves any error when i paste `https://snapshot.box` or  `http://httpstat.us/204`

<img width="478" height="93" alt="image" src="https://github.com/user-attachments/assets/de3bc2d2-2426-4792-8dd9-623af5840874" />